Stages.md

```diff
@@ -0,0 +1,125 @@
+
+# Details
+
+This document formalizes some signal checkpoints for new features in existing WHATWG specs. These checkpoints are modeled loosely on the [TC39 process](https://tc39.es/process-document/), which uses the concept of _stages_. Each subsequent stage implies a larger degree of consensus from the community, willingness to engage, and/or intent to implement and ship the feature in browser engines, and signals progress to web developers.  These checkpoints are not required for all features; this is simply a way to ensure that attention is appropriately drawn to incubations that have grown community support behind them.
+
+Terminology:
+
+* _Stage_: each successive stage is closer to a landed spec and interoperable implementation.  These stages are tracked on the issues in the relevant WHATWG Github repository, using labels ("stage 0", "stage 1", etc.)
+* _Browser engine_: an independent implementation of the web platform (e.g. Chromium, WebKit and Gecko at present).
+* _Implementation interest_: a direct signal from a browser engine that a feature is on the roadmap to implement, as defined in [working mode](https://whatwg.org/working-mode#additions)].
+* _Browser engine representatives_: persons delegated by a browser engine to indicate implementation interest.
+* _Proposal champion_: person/organization that proposed and will advance a feature through the stages, typically including writing the specification, researching use cases, and prototyping.
+
+Process:
+
+* Any proposal effectively starts at Stage 0 without any approvals, by a community member filing a new issue on a relevant WHATWG spec.
+* In order to drive a proposal forward, it is expected that a community member will step up to champion the proposal, through answering questions asynchronously, providing feedback when requested, and consideration of important open questions at triage meetings.
+* Stage level is tracked with labels on issues in the relevant specifications — “stage 0”, “stage 1”, etc.
+* Stage labels (other than stage 0) should only be added by browser engine representatives, or by Editors on the relevant specification, after showing the necessary support for a level.
+* **Advancing stages**
+    * The feature champion should bring the proposal to the WHATWG to advance to the next stage by setting an “Agenda+” label on the tracking issue, and showing up to (or ensuring someone will show up to) the next triage meeting to discuss. Advancing to a new stage requires support for the decision by at least two implementers (via their browser engine representatives), and there should not be any strong implementer objections ([per the working mode](https://whatwg.org/working-mode#additions)). This support can also be gathered in any public manner, e.g. GitHub issue comments, triage meetings, etc. 
```

Comment:
The first sentence about agenda+ and triage meetings seems to contradict the last sentence about gathering support in any manner. Maybe it can be deleted?

Reply:
Although I agree the last sentence is not entirely necessary, it's not contradictory - the first sentence is about "how do you bring the proposal to the group", the last sentence is "how do you demonstrate support from implementers".